### DISCUSSION TITLE
Google play release - Problem with clones?

### DISCUSSION THREAD

**Discussion by inson1** [FossifyOrg/General-Discussion#48] [Q&A] [answered]
Google has problems with clones. Could there be problem? What is still clone and what isnt? (I will then add it to the Readme).

**Comment by naveensingh** [comment]
There is a repetitive content policy: https://support.google.com/googleplay/android-developer/answer/9899034?hl=en#zippy=%2Cexamples-of-common-violations. We'll see.

**Comment by And96** [comment]
> Google has problems with clones. Could there be problem? What is still clone and what isnt? (I will then add it to the Readme).

Changing name, icon and package I think will them available. Since next releases they will start to have differences so... I think it's not a problem. Will see

**Comment by naveensingh** [comment, marked as answer]
Fossify File Manager is now publicly available so I don't think they have a problem with our forks. https://github.com/orgs/FossifyOrg/discussions/57#discussioncomment-8262117
